hack/make/.integration-test-helpers

```diff
@@ -33,6 +33,10 @@ run_test_integration() {
 }
 
 build_test_suite_binaries() {
+	if [ $DOCKER_INTEGRATION_TESTS_VERIFIED ]; then
```

Comment:
nit: this variable read to me like "the tests have run and passed" but I guess it's fine